test/NuGet.Core.Tests/NuGet.ProjectModel.Test/NjJsonPackageSpecReaderTests.cs

```diff
@@ -20,7 +21,8 @@
 namespace NuGet.ProjectModel.Test
 {
     [UseCulture("")] // Fix tests failing on systems with non-English locales
-    public class JsonPackageSpecReaderTests
+    [Obsolete]
+    public class NjJsonPackageSpecReaderTests
```

Comment:
As I mentioned in another comment about environment variables, I think it would be better to have a generic `JsonPackageSpecReaderTests` , but modify all the tests to be `[Theory]`, have the method take a `bool useNewtonsoftJson`, then have a single test class test both the Newtonsoft.Json and System.Text.Json code paths. This will make it much easier to ensure that both produce the same outputs for the same inputs. This will be particularlyl important when there are future changes to the assets file schema, as it might be too easy to accidentally change only one of the two parsers, and then customers using the environment variable have a bug.